packages/opensrp-server-service/src/serviceClass.ts

```diff
@@ -126,7 +126,7 @@ export class OpenSRPService<PayloadT extends object = Dictionary> {
    */
   public static getURL(generalUrl: string, params: paramsType): string {
     if (params) {
-      return `${generalUrl}?${queryString.stringify(params)}`;
+      return `${generalUrl}?${queryString.unescape(queryString.stringify(params))}`;
```

Comment:
unescape() is not recommended to use, it is kind of deprecated for latest websites. not sure if it is working or not.
You can use decodeURI() or decodeURIComponent() instead if that meets the requirement.